src/test/java/org/ibp/api/java/impl/middleware/ontology/validator/VariableValidatorTest.java

```diff
@@ -303,19 +304,22 @@ public void testWithSpecialCharacterAnsStartWithDigitVariableName() throws Middl
 	}
 
 	/**
-	 * Test for Alias contain special character and start with digit
+	 * Test for variables with the same Name
```

Comment:
rename tests with a more significant name, describing the test case @cuenyad

Reply:
done.